tests/test_biasadjust.py

```diff
@@ -292,15 +293,15 @@ def test_xclim_vs_xscen(
             dhist,
             var="pr",
             period=["2001", "2003"],
-            adapt_freq={"thresh": "1 mm d-1"},
+            adapt_freq={"thresh": "1 kg m-2 d-1"},
             xclim_train_args={"kind": "*", "nquantiles": 50},
```

Comment:
This is because there is no context="infer" conversion in `xsdba` to remain climate agnostic